packages/bundle-size-checker/src/renderMarkdownReport.js

```diff
@@ -66,69 +66,66 @@ function generateEmphasizedChange({ id: bundle, parsed, gzip }) {
-      Math.abs(gzip.absoluteDiff) > gzipSizeChangeThreshold;
-    if (isSignificantChange && visibleEntries.length < visibleLimit) {
-      visibleEntries.push(entry);
-    } else {
-      hiddenEntries.push(entry);
-    }
+  if (track.length > 0) {
+    const entryMap = new Map(comparison.entries.map((entry) => [entry.id, entry]));
+    const trackedEntries = track.map((bundleId) => {
+      const trackedEntry = entryMap.get(bundleId);
+      if (!trackedEntry) {
+        throw new Error(`Tracked bundle not found in head snapshot: ${bundleId}`);
+      }
+      return trackedEntry;
+    });
+    // Show all tracked bundles directly (including unchanged ones)
+    const trackedChanges = trackedEntries.map(generateEmphasizedChange);
+    markdownContent += `${trackedChanges.join('\n')}`;
+  } else {
+    markdownContent += `**Total Size Change:** ${formatChange(
+      comparison.totals.totalParsed,
+      comparison.totals.totalParsedPercent,
+    )} - **Total Gzip Change:** ${formatChange(
+      comparison.totals.totalGzip,
+      comparison.totals.totalGzipPercent,
+    )}\n`;
+
+    markdownContent += `Files: ${comparison.fileCounts.total} total (${
+      comparison.fileCounts.added
+    } added, ${comparison.fileCounts.removed} removed, ${comparison.fileCounts.changed} changed)\n\n`;
   }
 
-  const importantChanges = visibleEntries.map(generateEmphasizedChange);
-  const hiddenChanges = hiddenEntries.map(generateEmphasizedChange);
-
-  // Add important changes to markdown
-  if (importantChanges.length > 0) {
-    // Show the most significant changes first, up to the visible limit
-    const visibleChanges = importantChanges.slice(0, visibleLimit);
-    markdownContent += `${visibleChanges.join('\n')}`;
-  }
-
-  // If there are more changes, add them in a collapsible details section
-  if (hiddenChanges.length > 0) {
-    markdownContent += `\n<details>\n<summary>Show ${hiddenChanges.length} more bundle changes</summary>\n\n`;
-    markdownContent += `${hiddenChanges.join('\n')}\n\n`;
+  // Show all entries in details section, not just changed ones
+  // Filter out tracked bundles to avoid duplication
+  const trackedIdSet = new Set(track);
+  const detailsEntries = comparison.entries.filter((entry) => !trackedIdSet.has(entry.id));
+
+  // Cap at maxDetailsLines bundles to avoid overly large reports
+  const cappedEntries = detailsEntries.slice(0, maxDetailsLines);
+  const hasMore = detailsEntries.length > maxDetailsLines;
+
+  if (cappedEntries.length > 0) {
+    const allChanges = cappedEntries.map(generateEmphasizedChange);
+    const bundleWord = cappedEntries.length === 1 ? 'bundle' : 'bundles';
+    const summaryText = hasMore
+      ? `Show details for ${cappedEntries.length} more ${bundleWord} (${detailsEntries.length - maxDetailsLines} more not shown)`
+      : `Show details for ${cappedEntries.length} more ${bundleWord}`;
+    markdownContent += `<details>\n<summary>${summaryText}</summary>\n\n`;
```

Comment:
A markdown table formatting could probably help the readability. I kind of feel that only the link to the web view is readable. But maybe we don't even need to list the details in markdown. Only having the web view seems fine. So maybe to delete in the first place.